docker/env_file_app_ci

```diff
@@ -30,6 +30,7 @@ ELASTICSEARCH_NO_OF_REPLICAS=0
 
 # analyzers secrets (optional)
 IPINFO_KEY=test
+CRIMINALIP_KEY=test
```

Comment:
this should match the name of the secrets that you put on the analyzer_config.json